gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradableObject.java

```diff
@@ -72,6 +72,10 @@ public abstract class GradableObject implements Serializable {
     @ToString.Include
     protected String name;
 
+    @Column(name = "GRADABLE_REFERENCE")
+    @ToString.Include(name = "gradable_reference")
+    protected String gradableReference;
```

Comment:
`GradableObject.gradableReference` seems a little redundant

```suggestion
    @Column(name = "REFERENCE")
    @ToString.Include(name = "reference")
    protected String reference;
```

Reply:
I suppose so. I just thought it was clearer. But you're probably right.